edb/server/compiler/compiler.py

```diff
@@ -2328,6 +2331,42 @@ def _get_data_mending_desc(
         )
 
 
+def _check_force_database_error(
+    ctx: CompileContext,
+    ql: qlast.Base,
+) -> None:
+    if isinstance(ql, qlast.ConfigOp):
+        return
+
+    try:
+        err = json.loads(_get_config_val(ctx, 'force_database_error'))
```

Comment:
Will this deserialize JSON on every compile?

It's not that show, just unnecessary.

Reply:
And `json.loads` throws an error if it gets None